test/suites/resources.sh

```diff
@@ -10,3 +10,47 @@ test_resources() {
   echo "${RES}" | grep -q "Used:"
   echo "${RES}" | grep -q "Total:"
 }
+
+test_resources_bcache() {
+  # Create two loop devices used for the bcache cache and backing device.
+  configure_loop_device loop_file_1 loop_device_1
+  configure_loop_device loop_file_2 loop_device_2
+
+  # Create bcache device.
+  # shellcheck disable=SC2154
+  make-bcache -C "${loop_device_1}" -B "${loop_device_2}"
+
+  # Register bcache device.
+  # shellcheck disable=SC2154
+  echo "${loop_device_1}" > /sys/fs/bcache/register
+  # shellcheck disable=SC2154
+  echo "${loop_device_2}" > /sys/fs/bcache/register
+
+  # Print for debugging purposes.
+  lxc query /1.0/resources | jq '.storage.disks'
+
+  # Check the bcache device is returned by LXD.
+  [ "$(lxc query /1.0/resources | jq -r '.storage.disks.[] | select(.id == "bcache0")')" != "" ]
+
+  # Get the bcache cache and backing devices.
+  cache_device_base="$(basename "${loop_device_1}")"
+  backing_device_base="$(basename "${loop_device_2}")"
+  cache_device="$(cat "/sys/block/${cache_device_base}/dev")"
```

Comment:
```suggestion
  cache_device="$(< "/sys/block/${cache_device_base}/dev")"
```

I've replaced all those instances were a small file is read by `$(< ...)` to avoid forking. It's a tiny optimization but for consistency, would you mind doing it here too, please?

Reply:
Fixed. Also moved the `modprobe` part into the test itself, found it's better suited there, also in case you run the test suite locally and don't have the mod loaded.